nilearn/decoding/decoder.py

```diff
@@ -234,6 +287,9 @@ def _parallel_fit(
                     dummy_output = estimator.class_prior_
                 elif isinstance(estimator, DummyRegressor):
                     dummy_output = estimator.constant_
+
+            if isinstance(estimator, (RidgeCV, RidgeClassifierCV)):
+                param["alphas"] = estimator.alpha_
```

Comment:
isn't it a bit surprising to have the name `alphas`, but a single value? esp. since `alphas` corresponds to a list everywhere else. maybe it should be a different name like `best_alpha`, or `alpha`?